modisco/util.py

```diff
@@ -474,9 +474,14 @@ def compute_per_position_ic(ppm, background, pseudocount):
     assert len(ppm.shape)==2
     assert ppm.shape[1]==len(background),\
             "Make sure the letter axis is the second axis"
-    assert (np.max(np.abs(np.sum(ppm, axis=1)-1.0)) < 1e-5),(
-             "Probabilities don't sum to 1 along axis 1 in "
-             +str(ppm)+"\n"+str(np.sum(ppm, axis=1)))
+    if (np.max(np.abs(np.sum(ppm, axis=1)-1.0)) < 1e-5):
```

Comment:
This should be a greater sign. A cleaner way, also:
`if not np.allclose(np.sum(ppm, axis=1), 1.0, atol=1.0e-5):`